### PR TITLE
WRN-3841: Added VideoPlayer unit tests and migrated ToggleButton unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for VideoPlayer and ToggleButton components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3841

### CODE REVIEW

File: ToggleButton/tests/ToggleButton-specs.js [hunk 1]
```diff
@@ -10,81 +12,81 @@ describe('ToggleButton', () => {
 	test(
 		'should use \'toggleOffLabel\' if toggled off and label provided',
```
Comment (alexandrumorariu) [inline ToggleButton/tests/ToggleButton-specs.js]: Please write the test(...) on a single line(same below).

File: ToggleButton/tests/ToggleButton-specs.js [hunk 2]
```diff
@@ -10,81 +12,81 @@ describe('ToggleButton', () => {
...
 				<ToggleButton toggleOffLabel={toggleOffLabel}>
 					{textChild}
 				</ToggleButton>
 			);
+			const button = screen.getByRole('button');
 
-			const button = toggleButton.find('Button');
 			const expected = toggleOffLabel;
-			const actual = button.text();
+			const actual = button.children[2].children[0].children[0];
```
Comment (alexandrumorariu) [inline ToggleButton/tests/ToggleButton-specs.js]: I do believe that you do not need .children[2].children[0].children[0].

File: ToggleButton/tests/ToggleButton-specs.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import ToggleButton from '../ToggleButton';
 
 describe('ToggleButton', () => {
```
Comment (alexandrumorariu) [inline ToggleButton/tests/ToggleButton-specs.js:6]: Remove the empty line between describe and the const declarations.

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**